README.md

```diff
@@ -380,6 +461,59 @@ may have the following properties:
     progress messages.
 
 
+### Daemon mode
+
+Test case files for the Daemon mode have the same fields as for Standalone mode
+with the following changes/additions
+
+Additional fields:
+
+* `input_plugin`: The unique [ID](https://www.elastic.co/guide/en/logstash/7.10/plugins-inputs-file.html#plugins-inputs-file-id)
+  of the input plugin in the tested configuration, where the test input is
+  coming from. This is necessary, if a setup with multiple inputs is tested,
+  which either have different codecs or are part of different pipelines.
+* `export_metadata`: Controls if the metadata of the event processed by Logstash
+  is returned. The metadata is contained in the field `[@metadata]` in the
+  Logstash event. If the metadata is exported, the respective fields are
+  compared with the expected result of the testcase as well. (default: false)
+* `export_outputs`: Controls if the ID of the output, a particular event has
+  emitted by, is kept in the event or not. If this is enabled, the expected
+  event needs to contain a field named `_lfv_out_passed` which contains the ID
+  of the Logstash output.
+* `testcases`:
+  * `event`: Local fields, only added to the events of this test case. These
+    fields overwrite global fields.
```

Comment:
Wouldn't it make more sense to call this option `fields` if the functionality indeed is the same as the global `fields` option?

Reply:
My idea was, that in `event` one provides the complete event with all fields set to the correct values needed for the test case, where `fields` (in the global section) is more about providing some fields, that are added to the input (the input line). That being said, I am fine with `fields` in the `testcases` section as well, so I have changed it.